js/app.js

```diff
@@ -10,11 +10,12 @@ navLinks.classList.toggle('active');
 logo.classList.toggle('active');
 });
 
-document.querySelector('.nav-link')
+document.querySelector('.nav-links')
 .forEach(n=>n.addEventListener('click', ()=>{
 hamburger.classList.remove('active');
 navLinks.classList.remove('active');
 logo.classList.remove('active');
+menu.classList.remove('active');
 // logo.style.display="none";
```

Comment:
- [ ] When I click on any nav link it takes me to the correct section of the page but the mobile menu still exists, so Kindly make the mobile menu disappear when you click on any nav link.

![chrome-capture-2022-4-18 (1)](https://user-images.githubusercontent.com/68796232/169033049-03447c48-9bb5-4d26-bf05-9f17433c22f7.gif)